wire-ios-data-model/sourcery/generated/AutoMockable.generated.swift

```diff
@@ -0,0 +1,35 @@
+// Generated using Sourcery 1.8.1 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
I would also think about modifying `.gitattributes` to not show changes for autogenerated files in git diff for pull requests, like `*.generated.swift linguist-generated=true`

Reply:
So we wouldn't see the diff in PRs but we would in git?

Reply:
You would see but it wouldn't be expanded by default

Reply:
ok cool, how can I make the required changes?

Reply:
add `*.generated.swift linguist-generated=true` to `.gitattributes` file 😄

Reply:
ok, figured it out :)